src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,12 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        String address;
+        if (isPrivate) address = "(private)";
```

Comment:
isPrivate case should be omitted here. Just return "Address: " + address is fine.